035-Circular_primes.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+'''
+The number, 197, is called a circular prime because all rotations of the digits: 197, 971, and 719, are themselves prime.
+
+There are thirteen such primes below 100: 2, 3, 5, 7, 11, 13, 17, 31, 37, 71, 73, 79, and 97.
+
+How many circular primes are there below one million?
+'''
+
+def is_prime(n):
+    if n == 2:
+        return True
+    elif n % 2 == 0 or n < 2:
+        return False
+    else:
+        # only odd numbers
+        for i in range(3, int(n ** .5) + 1, 2):
+            if n % i == 0:
+                return False
+    return True
+
+
+def rotation(n):
+    N = []
```

Comment:
Use meaningful var names. N doesn't mean anything to someone who looks at this code

Reply:
changed